clinvar-variant-types/clinvar-variant-types.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
+
+import argparse
+import gzip
+import xml.etree.ElementTree as ElementTree
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--clinvar-xml', required=True)
+args = parser.parse_args()
+
+
+def add_transitions(transitions_dict, transition_chain):
+    """Increments the count of a particular flow in Sankey diagram."""
+    for transition_from, transition_to in zip(transition_chain, transition_chain[1:]):
+        transitions_dict.setdefault((transition_from, transition_to), 0)
+        transitions_dict[(transition_from, transition_to)] += 1
+
+
+# The dicts store transition counts for the Sankey diagrams. Keys are (from, to), values are transition counts.
+# Sankey diagrams can be visualised with SankeyMatic.
```

Comment:
maybe add quick instructions to run SankeyMatic? I guess the url http://www.sankeymatic.com/build/ is fine.

Reply:
Thanks, added in https://github.com/EBIvariation/eva-opentargets/pull/126/commits/2d403ef6e570aa518628ee9e53a806e68d09d065